packages/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -0,0 +1,304 @@
+<template>
+  <div
+    v-if="isVisible"
+    :aria-label="title"
+    class="k-modal-fullscreen isOpen"
+    role="dialog"
+    aria-modal="true">
+    <div
+      class="k-modal-dialog"
+    >
+      <div class="k-modal-header">
+        <div
+          v-if="!$scopedSlots.title"
+          class="k-modal-header-description mb-5"
+          role="heading"
+          aria-level="2">
+          <div class="k-modal-title">
+            <span
+              class="header-icon">
+              <KIcon
+                :icon="iconHeader"
+                class="mr-2" />
+            </span>
+            <span class="header-content">
+              <slot name="header-content">{{ title }}</slot>
+            </span>
+          </div>
+          <div class="k-modal-action ml-3">
+            <slot name="action-buttons">
+              <KButton
+                appearance="outline"
+                class=" mr-2"
+                @click="close">
+                {{ cancelButtonText }}
+              </KButton>
+              <KButton
+                :appearance="type === 'danger' ? 'danger' : 'primary'"
+                @click="proceed">
+                {{ actionButtonText }}
+              </KButton>
+            </slot>
+          </div>
+        </div>
+        <div>
+          <hr>
+        </div>
+      </div>
+      <div class="k-modal-body-description">
+        <slot name="body-header-description">{{ bodyHeaderDescription }}</slot>
+      </div>
+      <div class="k-modal-body">
+        <slot name="body-content">{{ content }}</slot>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import KButton from '@kongponents/kbutton/KButton.vue'
+import KIcon from '@kongponents/kicon/KIcon.vue'
+
+export default {
+  name: 'KModalfullscreen',
+  components: { KButton, KIcon },
+
+  props: {
+    /**
+     * Set the text of the title, if using title slot
+     */
+    title: {
+      type: String,
+      required: true
+    },
+    /**
+     * Text to display Page title and description
+     */
+    bodyHeaderDescription: {
+      type: String,
+      default: ''
+    },
+    type: {
+      type: String,
+      default: 'info',
+      validator: (value) => {
+        return [
+          'info',
+          'warning',
+          'danger'
+        ].includes(value)
+      }
+    },
+    /**
+     * Set the text of the body content
+     */
+    content: {
+      type: String,
+      default: ''
+    },
+    /**
+      *  Pass whether or not the modal should be visible
+      */
+    isVisible: {
+      type: Boolean,
+      default: false
+    },
+    /**
+     * Set the text of the close/cancel button
+     */
+    cancelButtonText: {
+      type: String,
+      default: 'Cancel'
+    },
+    actionButtonText: {
+      type: String,
+      default: 'Save'
+    },
+    /**
+     * Set the appearance of the close/cancel button
+     */
+    cancelButtonAppearance: {
```

Comment:
You don't seem to be using this prop, let's update the code to use it. Also can we add `actionButtonAppearance` prop?